docs/notebooks/image_to_image_arcface.md

```diff
@@ -0,0 +1,228 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: .venv
+    language: python
+    name: python3
+---
+
+<!-- #region id="p8jc8EyfruKw" -->
+# Image-to-Image Search with ArcFaceLoss
+
+<a href="https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg"></a>
+
+Using image queries to search for visually similar images is a very popular use case. However, pre-trained models do not deliver the best results. Models are trained on general data that lack knowledge related to your specific task. Here's where Finetuner comes in! It enables you to easily add task-specific knowledge to a model.
+
+Where our [previous guide](https://finetuner.jina.ai/notebooks/image_to_image/) showed off finetuning with `TripletMarginLoss`, 
+this guide will perform finetuning on a dataset with fewer classes, and more documents per class. To improve our performance in this case, we will use `ArcFaceLoss` as our loss function this time.
```

Comment:
```suggestion
this guide will perform fine-tuning on a dataset with fewer classes, and more documents per class. To improve our performance in this case, we will use `ArcFaceLoss` as our loss function this time.
```

Maybe it is better to say that arcface is good if (most of) the classes are known during the training. Though, I am not sure if it is true, but I think this might rather be the reason why arcface did not work on TLL.